packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -88,7 +88,7 @@ export default function EditorStyles( { styles, scope } ) {
 		return [
 			transformStyles(
 				_styles.filter( ( style ) => style?.css ),
-				scope
+				scope ? `:where(${ scope })` : undefined
```

Comment:
Just a non-blocking question: is it better to do this internally within `EditorStyles` or would there be any benefit of having the consumer pass in `:where(.editor-styles-wrapper)` explicitly for the scope?

Reply:
Given that the intention behind `scope` is to make sure there's no bleed of content styles into the editor chrome, I think it makes sense for the scoping selector to always have 0 specificity. This also helps ensure the specificity will always be the same between editor and front end.